Tooling-Landscape/Unanimous-Understanding/OSS_Tooling_Landscape_UMLComp.plantuml

```diff
@@ -59,6 +59,13 @@ node "Company Internal" <<Company Network>> {
 
     component [Security Vulnerability Assessment] as securityassessment <<Internal Service>>
 
+    component [Forensic Code Analyzer] as forensiccodeanalyzer <<Internal Service>>
+
+    component [Obligation Fulfillment Management] as obligationfulfillmentmgmt <<Internal Service>>
+    
+    obligationfulfillmentmgmt ..> forensiccodeanalyzer : use
```

Comment:
This one is wrong. This should be:
`obligationfulfillmentmgmt ..> fossartefactgenerator : use`